source/standards/monitoring.html.md.erb

```diff
@@ -1,44 +1,24 @@
 ---
-title: Monitoring
+title: Monitor your service
 expires: 2018-03-01
 ---
 
 # <%= current_page.data.title %>
 
 <%= partial :expires %>
 
-## Availability monitoring
+At GDS, we follow the principles set out in the Service Manual for [monitoring the status of services](https://www.gov.uk/service-manual/technology/monitoring-the-status-of-your-service) and for [setting performance metrics](https://www.gov.uk/service-manual/measuring-success/how-to-set-performance-metrics-for-your-service).
 
-[Pingdom](https://www.pingdom.com/) is the tool we've used most for availability
-monitoring. If you need to monitor a service from outside the US or EU,
-GOV.UK has used [CA App Synthetic Monitor](https://asm.ca.com/en/).
+As part of your monitoring and alerting strategy we recommend you:
 
-## Smoke tests
+* use [Pingdom](https://www.pingdom.com/) to monitor an EU or US-based service
+* use [CA App Synthetic Monitor](https://asm.ca.com/en/) to monitor a service from outside the EU or US
+* run regular [smoke tests](https://www.gov.uk/service-manual/technology/deploying-software-regularly#using-smoke-tests-after-you-deploy) using a browser automation app such as [Selenium](http://www.seleniumhq.org/)
+* implement a tool to ensure user journeys are working as expected
```

Comment:
we could do with an example of a tool we already use at GDS or recommend teams use, any suggestions?